### PR TITLE
introduced component state

### PR DESCRIPTION
**Pull request by Awthor**
Moved the movies data model to a proper front-end data source & Introduced component state  and tidied up code :)

### CODE REVIEW

File: src/Gallery/Movies-get.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Gladiator from './Gladiator.jpg';
+import LOTR from './LOTR.jpg';
+import PulpFiction from './Pulp Fiction.jpg';
+import StarWars from './Star Wars.jpg';
+import Terminator from './Terminator.jpg';
+import TopGun from './Top Gun.jpg';
+
+export default function getMovies() {
```
Comment (rmallols) [inline src/Gallery/Movies-get.js:8]: Could you please work a bit on the indentation and break line strategy?

```
[{ 
    name: "Gladiator", 
    picture: Gladiator, 
    id: 'Galdiator'
}, { 
    name: "Lord of The Ring", 
    picture: LOTR, 
    id: 'LordofTheRings' 
},
...
```

File: src/Pages/Details.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        this.state = {
+        message:
+        'Hello, this will be the details page for each Movie & TV show :)'
+    };
+}
+
+componentDidMount() {
+    setTimeout(() => {
+        this.state({
+        message: 'Coming soon! :)'
```
Comment (rmallols) [inline src/Pages/Details.js:16]: Wrong indentation here

File: src/Pages/Details.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        })
+    }, 3000);
+}
+
+render() {
+    return (
+        <div>
+            <h1>{this.state.message}</h1>
+        </div>
+        )
```
Comment (rmallols) [inline src/Pages/Details.js:26]: Missing semi-colon

Comment (rmallols) [inline src/Pages/Details.js:26]: Missing semi-colon

**Review by rmallols — CHANGES_REQUESTED** [review]

File: src/Gallery/Movies-get.js (hunk 1 above)
Comment (Awthor) [reply src/Gallery/Movies-get.js:8]: done.

File: src/Pages/Details.js (hunk 2 above)
Comment (Awthor) [reply src/Pages/Details.js:16]: done

File: src/Pages/Details.js (hunk 3 above)
Comment (Awthor) [reply src/Pages/Details.js:26]: done

**Pull request merged**